tests/gorepo_test.go

```diff
@@ -22,3 +25,21 @@ func TestGoRepositoryCompilerTests(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// Test that GopherJS can be vendored into a project, and then used to build Go programs.
+// See issue https://github.com/gopherjs/gopherjs/issues/415.
+func TestGopherJSCanBeVendored(t *testing.T) {
+	if runtime.GOARCH == "js" {
+		t.Skip("test meant to be run using normal Go compiler (needs os/exec)")
+	}
+
+	cmd := exec.Command("bash", "gopherjsvendored_test.sh")
```

Comment:
Did you see the commit message of 49f2b5ca9ae7415de29eb86c95d2cecaca9d606f?

However, I think I know what it'll take to make it work with `sh`. It might just be a matter of removing the `SIG` prefix from the signal names.

Reply:
Is it not cleaner to just write this test as `// +build ignore`-d `.go` file in any case? That way we don't need to worry about shells, traps etc.

Reply:
I often prefer Go over other tools, but a bash script seemed like a good fit for this task. It was based on the bash script in [playground repo](https://github.com/gopherjs/gopherjs.github.io/blob/b3fd139784228a7f0371aa48599b6377bac7fbaf/playground/update.sh), which has been used with success for some years.

It's already done and it works well. I don't see a need to spend much more time on it.

Reply:
Will leave it up to you. It might be, however, that addressing https://github.com/gopherjs/gopherjs/pull/787/files/2ad4a5b9a30e5f1f31c643c2a2ac27543a5ddb58#r181624969 would be easier/clearer in a `.go` file.